### PR TITLE
Add very basic usage example for android toast

### PR DESCRIPTION
**Pull request by mdamien**
Currently, it's hard to find out that it's very easy to use android toasts. The example is very complex and hard to grasp for a beginner. So, I propose to add an easy to understand example.

I'm sure the syntax of this PR is not really correct, it's just a start to show the kind of thing I propose.

### CODE REVIEW

**Comment by facebook-github-bot-8** [comment]
By analyzing the blame information on this pull request, we identified @donyu to be a potential reviewer.

**Comment by mdamien** [comment]
Also, I didn't check if the `Ctrl-R` shorcut actually work in the simulator. It may be good to tell about the device-shaking / debug menu also.

File: Libraries/Components/ToastAndroid/ToastAndroid.android.js [hunk 1]
```diff
@@ -22,6 +22,12 @@ var RCTToastAndroid = require('NativeModules').ToastAndroid;
  *
  * There is also a function `showWithGravity` to specify the layout gravity. May be
  * ToastAndroid.TOP, ToastAndroid.BOTTOM, ToastAndroid.CENTER
+ *
```
Comment (donyu) [inline Libraries/Components/ToastAndroid/ToastAndroid.android.js]: You'll want to add a period after this last comment.

File: Libraries/Components/ToastAndroid/ToastAndroid.android.js [hunk 2]
```diff
@@ -22,6 +22,12 @@ var RCTToastAndroid = require('NativeModules').ToastAndroid;
  *
  * There is also a function `showWithGravity` to specify the layout gravity. May be
  * ToastAndroid.TOP, ToastAndroid.BOTTOM, ToastAndroid.CENTER
+ * 
+ * Basic usage:
+ * ```javascript
+ * import { ToastAndroid } from 'react-native';
```
Comment (donyu) [inline Libraries/Components/ToastAndroid/ToastAndroid.android.js]: Importing this module should be obvious.

Comment (satya164) [inline Libraries/Components/ToastAndroid/ToastAndroid.android.js]: Yeah, please remove this import statement

File: Libraries/Components/ToastAndroid/ToastAndroid.android.js [hunk 3]
```diff
@@ -22,6 +22,12 @@ var RCTToastAndroid = require('NativeModules').ToastAndroid;
...
+ * ToastAndroid.show('A pikachu appeared nearby !', ToastAndroid.SHORT);
```
Comment (satya164) [inline Libraries/Components/ToastAndroid/ToastAndroid.android.js]: I'd move this example to before the note about `showWithGravity`, also maybe add an example for `showWithGravity` too.

**Comment by facebook-github-bot-5** [comment]
@mdamien updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9859/files/9b72bf9e95b68be642bf608a5a88daf67e957673..faeba996a76f11c86aa1a80796e26a96ccb204a9)

**Comment by facebook-github-bot-3** [comment]
@mdamien updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9859/files/faeba996a76f11c86aa1a80796e26a96ccb204a9..91ce7a4ee0019d535db2313896c2f5517ac78cdf)

File: Libraries/Components/ToastAndroid/ToastAndroid.android.js [hunk 4]
```diff
@@ -21,7 +21,13 @@ var RCTToastAndroid = require('NativeModules').ToastAndroid;
  * 2. int duration: The duration of the toast. May be ToastAndroid.SHORT or ToastAndroid.LONG
  *
  * There is also a function `showWithGravity` to specify the layout gravity. May be
- * ToastAndroid.TOP, ToastAndroid.BOTTOM, ToastAndroid.CENTER
+ * ToastAndroid.TOP, ToastAndroid.BOTTOM, ToastAndroid.CENTER.
+ *
+ * Basic usage:
+ * ```javascript
+ * ToastAndroid.show('A pikachu appeared nearby !', ToastAndroid.SHORT);
```
Comment (donyu) [inline Libraries/Components/ToastAndroid/ToastAndroid.android.js]: super nit: extra space between nearby and exclamation point.

**Comment by donyu** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-5** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3886274) to review internal test results.

**Pull request closed without merging**